### PR TITLE
gh: update contributing and readme files

### PR DESCRIPTION
**Pull request by sevki**
contributing is fine but the what is harvey section in read me needs help.

Signed-off-by: Sevki <s@sevki.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ Harvey: A distributed operating system
 Welcome to Harvey, we are delighted that you are interested in the
 project.
 
+## What is harvey?
+
+Harvey is a distributed operating system. It is direct descendant
```
Comment (rminnich) [inline README.md]: add "of"

File: README.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ Harvey: A distributed operating system
...
+[Plan 9 from Bell Labs](https://en.wikipedia.org/wiki/Plan_9_from_Bell_Labs) which was originally
+developed by the same group at Bell Labs that developed Unix and C.
```
Comment (rminnich) [inline README.md]: maybe change developed to invented? Not sure.

**Comment by rminnich** [comment]
It's fine. Just add the word of as suggested, the other parts are optional suggestions. 

Better to ship a good thing than never ship the perfect thing.

LGTM

**Comment by sevki** [comment]
going to wait to merge this one in for a little in case others want to add to this.

**Comment by elbing** [comment]
LGTM

**Pull request merged**